cluster/common.sh

```diff
@@ -678,6 +678,7 @@ PROMETHEUS_TO_SD_ENDPOINT: $(yaml-quote ${PROMETHEUS_TO_SD_ENDPOINT:-})
 PROMETHEUS_TO_SD_PREFIX: $(yaml-quote ${PROMETHEUS_TO_SD_PREFIX:-})
 ENABLE_PROMETHEUS_TO_SD: $(yaml-quote ${ENABLE_PROMETHEUS_TO_SD:-false})
 ENABLE_POD_PRIORITY: $(yaml-quote ${ENABLE_POD_PRIORITY:-})
+GCE_ALPHA_FEATURES: $(yaml-quote ${GCE_ALPHA_FEATURES:-})
```

Comment:
Take a look at PROVIDER_VARS, as we don't want to have cloud provider specific vars leak in to common.sh.

Reply:
Good point. Didn't know we had PROVIDER_VARS. Updated the PR